src/i18n/en_AU.ts

```diff
@@ -1148,9 +1148,9 @@ export default {
                 staffList: 'K-9',
             },
             {
-                caption: 'Police Motorcycle',
+                caption: 'Motor Officer',
```

Comment:
this is wrong here, the way it was before is right. 
The caption that appears in the school and in the course window of the alliance must be placed here.
https://cdn.discordapp.com/attachments/544869019673559060/918244105979588698/unknown.png